.github/workflows/build.yml

```diff
@@ -131,7 +131,7 @@ jobs:
     - name: install monero dependencies
       run: sudo apt -y install build-essential cmake libboost-all-dev miniupnpc libunbound-dev graphviz doxygen libunwind8-dev pkg-config libssl-dev libzmq3-dev libsodium-dev libhidapi-dev libnorm-dev libusb-1.0-0-dev libpgm-dev libprotobuf-dev protobuf-compiler
     - name: install requests
-      run: pip install requests
+      run: pip install requests psutil monotonic
```

Comment:
Why do you need this library? time.monotonic is a python built-in.

Reply:
I also wanted to use time.monotinic(), but the CI crashed. It turned out, that we were using Python2 all the time, hence:
https://github.com/monero-project/monero/pull/7396

Either we leave it like it is (Python2 code), or we quickly merge the https://github.com/monero-project/monero/pull/7396

Reply:
I haven't looked at the runner in detail, but these files have a shebang line which claims python3 is to be expected, so it's problematic to be using python2. I also note that the files import `print_function` from `__future__`, which would imply python2 is in use. Overall, I would try to clean things up to use python3, as python2 is EOL and dying rather quickly (the only reason it's still available, and sometimes the default in certain distros, is because core system components have assumed it, and it takes a while to make all the requisite changes, though I think most distros are quite close and it'll be gone in a year).